contrib/rpm/README.md

```diff
@@ -0,0 +1,185 @@
+RPM Spec File Notes
+-------------------
+
+The RPM spec file provided here is for Bitcoin-Core 0.12.0 and builds on CentOS
```

Comment:
Well my primary motivation was to provide something RPM packagers could use as a starting point that puts things in the proper places, does the proper testing on build of the binaries, etc.

RPM spec files distributed rarely make into any package repository without some tweaking by the packager. For example, Fedora packaging guidelines discourage packaging of the .la libtool files or the .a static libraries. So someone building for Fedora packaging would likely change `%install` to delete those files, and then remove them from the `%files bitcoin-devel`.

If preferred a `bitcoin.specfile.in` file could be created so that autoconf / automake system could fill in the version, that means autoconf has to be run before the spec file is useful, and it makes the `%changelog` less clear.